src/web/app/components/question-submission-form/question-submission-form.component.ts

```diff
@@ -91,13 +91,32 @@ export class QuestionSubmissionFormComponent implements OnInit {
   allowedToHaveParticipantComment: boolean = false;
 
   constructor(private feedbackQuestionsService: FeedbackQuestionsService,
-              private feedbackResponseService: FeedbackResponsesService) {
+    private feedbackResponseService: FeedbackResponsesService) {
     this.visibilityStateMachine =
-        this.feedbackQuestionsService.getNewVisibilityStateMachine(
-            this.model.giverType, this.model.recipientType);
+      this.feedbackQuestionsService.getNewVisibilityStateMachine(
+        this.model.giverType, this.model.recipientType);
   }
 
   ngOnInit(): void {
+    this.sortRecepientsByName(this.model);
+  }
+
+  /**
+   * Sorts recepients of feedback by their name.
+   *
+   * Feedback recepients are initially sorted by their email address.
+   */
+  private sortRecepientsByName(model: QuestionSubmissionFormModel): void {
+    model.recipientSubmissionForms.sort((firstRecepient: FeedbackResponseRecipientSubmissionFormModel,
+      secondRecepient: FeedbackResponseRecipientSubmissionFormModel) => {
+      const indexOne: number = model.recipientList.findIndex((x: FeedbackResponseRecipient) =>
```

Comment:
I'm not sure if `recipientList` is always in alphabetical order. I think it would be good to sort the `recipientList` first, you can put the sorting in this function as well.

Reply:
This was my thoughts exactly! I was a bit worried that sorting a long list of students every refresh could slow things down, and from what I've seen it was sorted, but sorting is definitely safer. Will make the changes